Content.Server/Traits/Assorted/UnrevivableSystem.cs

```diff
@@ -9,7 +9,7 @@ public override void Initialize()
     {
         base.Initialize();
 
-        SubscribeLocalEvent<UnrevivableComponent, CloningAttemptEvent>(OnCloningAttempt);
+       // SubscribeLocalEvent<UnrevivableComponent, CloningAttemptEvent>(OnCloningAttempt);
```

Comment:
Looks like I originally missed that this needs a harmony comment
```suggestion
       // SubscribeLocalEvent<UnrevivableComponent, CloningAttemptEvent>(OnCloningAttempt); Harmony
```